Tests/BaseObjectTests.swift

```diff
@@ -16,181 +18,183 @@ class BaseObjectTests: QuickSpec {
             }
 
             beforeEach {
-                baseObject = BaseObject()
+                baseObject = ConcreteBaseObject()
                 callbackWasCalled = false
             }
 
-            describe("on") {
-                it("Callback should be called on event trigger") {
-                    baseObject.on(eventName, callback: callback)
-                    baseObject.trigger(eventName)
-
-                    expect(callbackWasCalled) == true
+            context("initialization") {
+                it("doesn't initialize EventDispatcher twice") {
+                    expect(baseObject.eventDispatcher) == baseObject.eventDispatcher
                 }
+            }
 
-                it("Callback should receive userInfo on trigger with params") {
-                    var value = "Not Expected"
-                    baseObject.on(eventName) { userInfo in
-                        value = userInfo?["new_value"] as! String
+            context("events") {
+                describe("on") {
```

Comment:
According to the [Project conventions](https://github.com/globoi/steve/wiki/Test-Conventions), we should add the # prefix on the Method Name.
```swift
context("#on")
```